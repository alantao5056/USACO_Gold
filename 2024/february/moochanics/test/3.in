10
2
332348569330293272 597084119029013128
413178159 604577006
2
692473305568866083 983077699718157119
150314271 434392576
2
424315254347892037 859914598155788951
735525700 760283955
2
298822019915285785 976626372353203562
343683093 157436909
2
73399382515887397 526237010162810243
720726937 916205013
2
661907204050099020 672695737906082573
602185788 246732147
2
692344014179578169 866549976013105258
149886438 314673302
2
33454357049389276 318772067591437910
854885827 848424588
2
153428188445854511 700748029026418195
39335860 526176834
2
322121574825232055 607148396715860804
118727598 390861029
