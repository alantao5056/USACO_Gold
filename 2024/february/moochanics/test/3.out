520234255 520234255
994017415 994017415
582426169 582426169
2705157845 2705157845
553276057 553276057
25417143 25417143
749983035 749983035
335015519 335015519
1935658903 1935658903
1118654565 1118654565
